docs/notebooks/deep_ensembles.pct.py

```diff
@@ -126,40 +125,29 @@ def build_cubic_model(data: Dataset) -> DeepEnsemble:
 test_points = tf.linspace(-6, 6, 1000)
 
 # generating a plot with ground truth function, mean prediction and 3 standard
-# deviations around it
-plt.scatter(inputs, outputs, marker=".", alpha=0.6, color="red", label="data")
-plt.plot(
-    test_points, objective(test_points, False), color="blue", label="function"
-)
+# deviations around it 
+plt.scatter(inputs, outputs, marker='.', alpha=0.6, color = 'red', label='data')
+plt.plot(test_points, objective(test_points, False), color='blue', label='function')
 y_hat, y_var = model.predict(test_points)
 y_hat_minus_3sd = y_hat - 3 * tf.math.sqrt(y_var)
 y_hat_plus_3sd = y_hat + 3 * tf.math.sqrt(y_var)
-plt.plot(test_points, y_hat, color="gray", label="model $\mu$")
-plt.fill_between(
-    test_points,
-    tf.squeeze(y_hat_minus_3sd),
-    tf.squeeze(y_hat_plus_3sd),
-    color="gray",
-    alpha=0.5,
-    label="$\mu -/+ 3SD$",
-)
+plt.plot(test_points, y_hat, color='gray', label='model $\mu$')
+plt.fill_between(test_points, tf.squeeze(y_hat_minus_3sd), tf.squeeze(y_hat_plus_3sd), color='gray', alpha=0.5, label='$\mu -/+ 3SD$')
```

Comment:
How do you format the notebooks? If you just run `black` it should ignore them. If you run `task format` it should format them to width 80 (which looks better in the docs as there's no scrolling).

Reply:
(assuming you run this from the root directory)

Reply:
oops, yes, I usually run only black :)